src/Analyzers/Core/Analyzers/UseNullPropagation/AbstractUseNullPropagationDiagnosticAnalyzer.cs

```diff
@@ -18,24 +18,40 @@ internal static class UseNullPropagationConstants
         public const string WhenPartIsNullable = nameof(WhenPartIsNullable);
```

Comment:
this file is for the existing feature, and just represents cleanup to help make its logic more shareable.  The new functionality is added to a sibling part.